apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -34,19 +34,24 @@ target_compile_definitions(halide_benchmarks PRIVATE -DUSE_HALIDE ${MISC_DEFINIT
 target_link_libraries(halide_benchmarks PRIVATE halide_blas)
 list(APPEND BLAS_NAMES halide)
 
-find_package(Eigen3 QUIET)
-if (NOT Eigen3_FOUND)
+find_package(Eigen3 QUIET NO_MODULE)
```

Comment:
Would be nice to add some comments here for future readers, since this took some back and forth to get right...

Reply:
Added comments summarizing our discussion.